src/convertToObject.js

```diff
@@ -6,7 +6,22 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const arr = sourceString
+    .split(';')
+    .map((el) => el.trim())
+    .filter((el) => el !== '');
+  const nweArr = arr.map((el) => el.split(':').map((item) => item.trim()));
```

Comment:
Avoid using variable names like 'nweArr'. It decreases code readability and seems to be a typo. Consider using a more descriptive name.